src/main/java/emissary/server/mvc/adapters/RequestUtil.java

```diff
@@ -73,6 +74,41 @@ public static String[] getParameterValues(final ServletRequest request, final St
         return retArray;
     }
 
+    /**
+     * Get attribute or parameters from request and return a list of Strings. Attribute has priority over parameter when
+     * both are present
+     */
+    public static List<String> getParameterValuesStringList(final ServletRequest request, final String param) {
```

Comment:
I'm leaning towards not including this method.

Reply:
That was my original reaction but I had not gone back to confirm history. If you both agree this would be unused then I would support removing this method as well.

Reply:
Only currently referenced by its own unit test. +1 for removing it.